x/ibc-rate-limit/contracts/rate-limiter/src/contract.rs

```diff
@@ -0,0 +1,501 @@
+#[cfg(not(feature = "library"))]
+use cosmwasm_std::entry_point;
+use cosmwasm_std::{
+    to_binary, Addr, Binary, Deps, DepsMut, Env, MessageInfo, Response, StdResult, Timestamp,
+};
+use cw2::set_contract_version;
+
+use crate::error::ContractError;
+use crate::management::{
+    add_new_channels, try_add_channel, try_remove_channel, try_reset_channel_quota,
+};
+use crate::msg::{ExecuteMsg, InstantiateMsg, QueryMsg};
+use crate::state::{ChannelFlow, FlowType, CHANNEL_FLOWS, GOVMODULE, IBCMODULE};
+
+// version info for migration info
+const CONTRACT_NAME: &str = "crates.io:rate-limiter";
+const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
```

Comment:
oh huh, is this standard things you have to do for cosmwasm?

Reply:
yeah, this is part of [cw2](https://docs.cosmwasm.com/cw-plus/0.9.0/cw2/spec) and is used for migrating the contract. At the moment, we don't have migrations, so we could remove it; just left it there since it's standard in most contracts.

Reply:
Re: Not having a migration entrypoint: My thought was that since any changes would have to be approved by governance anyway, it would be easier to just deploy a new contract and update the param to point to it. 

Now that I think about it again, though, I'm noticing this would require two transactions (instantiate a new contract to get its address, update the param) instead of one (migrate the existing contract). So I should probably add a migration entrypoint

Reply:
Added a migration entrypoint in case we want to add migrations in the future